src/com/mambu/apisdk/util/MambuEntityType.java

```diff
@@ -31,7 +32,7 @@
 public enum MambuEntityType {
 
 	// A list of entities supported by the API wrapper library. To be extended as needed
-	CLIENT(Client.class), GROUP(Group.class), LOAN_ACCOUNT(LoanAccount.class), SAVINGS_ACCOUNT(SavingsAccount.class), LOAN_PRODUCT(
+	CLIENT(Client.class), CLIENT_EXTENDED(ClientExpanded.class), GROUP(Group.class), LOAN_ACCOUNT(LoanAccount.class), SAVINGS_ACCOUNT(SavingsAccount.class), LOAN_PRODUCT(
```

Comment:
class name is ClientExpanded, I think we should name the entity type in the same way